library/utils.c

```diff
@@ -166,6 +166,24 @@ int ziti_log_level() {
     return ziti_log_lvl;
 }
 
+const char* ziti_log_level_label() {
+    return level_labels[ziti_log_lvl];
+}
+
+static int get_debug_level(char* log_level) {
```

Comment:
I would not change the level if invalid value was passed in

Reply:
done. I made a small correction in the for loop. log_levels[] does not have NULL as the last element. So I changed the condition check to i < 7

Reply:
Instead of hard-coding the loop to 7 iterations, it would be better to use the `level_labels` array:

```
int num_levels = sizeof(level_labels) / sizeof(const char *);
```